google/cloud/logging/_helpers.py

```diff
@@ -50,6 +54,9 @@ class LogSeverity(object):
     logging.NOTSET: LogSeverity.DEFAULT,
 }
 
+_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%f%z"
```

Comment:
Does this need to be kept private? Might be useful for folks who are constructing their own string.

Reply:
Maybe, but I'd prefer to be conservative about what we expose publicly. I'd prefer to wait to see if there's demand, or I understand the problem space a bit better